tests/storage/test_storage.py

```diff
@@ -659,6 +672,16 @@ def test_store_and_fetch_datepath(self):
             path = store.make_key_path(namespace="x", filepath="test.pdf")
             assert store.fetch_file_contents(path) == b"hello"
 
+    def test_get_last_modified(self):
+        with tempfile.TemporaryDirectory() as tdir:
+            store = storage.LocalFileStore("bucket", tdir, use_date_in_key_path=True)
+
+            __, path = store.store_file(namespace="x", filename="test.pdf", contents="hello")
+
+            last_modified = store.get_last_modified(path)
+            assert last_modified is not None
+            assert type(last_modified) == datetime.datetime
```

Comment:
♻️ Don't feel like we need to do type checking here.

Reply:
Same as above about being redundant.